### PR TITLE
TFKUBE-401: Follow up for jira snapshot restore

### PR DESCRIPTION
**Pull request by nghazali**
## Pull request description

_Provide description for the PR_

## Checklist
- [x] I have successful end to end tests run (with & without domain)
- [x] I have added unit tests (if applicable)
- [x] I have user documentation (if applicable)

### CODE REVIEW

File: modules/AWS/rds/locals.tf [hunk 1]
```diff
@@ -1,5 +1,7 @@
 locals {
-  product_db_name    = var.product_db_name == null ? var.product : var.product_db_name
+  product_db_name   = var.product_db_name == null ? var.product : var.product_db_name
```
Comment (yzha645) [inline modules/AWS/rds/locals.tf:2]: What is `local.product_db_name` used for? Seems like `var.product_db_name` never get passed in via jira module...

Comment (nghazali) [reply modules/AWS/rds/locals.tf]: I cleaned it up.

File: modules/AWS/rds/locals.tf [hunk 2]
```diff
@@ -1,5 +1,7 @@
 locals {
-  product_db_name    = var.product_db_name == null ? var.product : var.product_db_name
+  product_db_name   = var.product
```
Comment (yzha645) [inline modules/AWS/rds/locals.tf:2]: nit: is it necessary to create a new variable for it? but I guess it clearly states that it's the db name.

File: modules/AWS/rds/locals.tf [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
+  rds_instance_name = var.snapshot_identifier == null ? var.product : null
```
Comment (yzha645) [inline modules/AWS/rds/locals.tf:3]: `rds_instance_name` will make me think it's `module.rds.identifier`. I would suggest naming the variable `db_name`.

**Review by jjeongatl — APPROVED** [review]

File: modules/AWS/rds/locals.tf (hunk 3 above)
Comment (nghazali) [reply modules/AWS/rds/locals.tf:3]: I renamed `rds_instance_id` to `rds_instance_indentifier` in bitbucket snapshot PR to make the variable name more clear. Considering that change this name will not be confused anymore.

**Pull request merged**